BUILD.md

```diff
@@ -82,10 +96,18 @@ Install [MSYS2] following its website's instructions and open a new
 "MSYS2 MSYS" window.  We'll need its collection of free programs and
 libraries.
 
-Install [gcc] and [libargp][argp] using MSYS2's `pacman` package
-manager:
+You can use the `pacman` package manager to install most of the tools
+needed to build Daffodil:
+
+    pacman -S clang diffutils git make pkgconf
```

Comment:
I wanted to show developers how to install nearly everything they might need for runtime2 development on each platform.  The only time you need diffutils is to run the c/Makefile's "make test" rule, which you'd do only by hand.  Actually you need [xmldiff](https://xmldiff.readthedocs.io/en/stable/installation.html) for that Makefile rule too but I decided not to mention xmldiff in BUILD.md because there's no MSYS2 package and the procedure's too complicated for most developers (install Python3 and pip, use pip to install xmldiff).  I also thought it would be too complicated to show developers how to build and run [include-what-you-use](https://github.com/include-what-you-use/include-what-you-use/blob/master/README.md), another tool that I use for updating the runtime2 C files' includes and comments showing what is used from each include.  Using iwyu requires some manual editing, so it's not easy to automate in CI either.